topics/icrc_29_window_post_message_transport.md

```diff
@@ -43,23 +44,32 @@ The signer should reply with a message indicating that it is ready to receive ad
 
 > **Note**: The relying party should send `icrc29_status` messages in short intervals. It is expected that some of the messages will be lost due to being sent before the signer is ready.
 
-After the `"result": "ready"` response has been received, the relying party can send [ICRC-25](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_25_signer_interaction_standard.md) messages to the signer
+After the `"result": "ready"` response has been received within a reasonable timeframe, the relying party can send [ICRC-25](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_25_signer_interaction_standard.md) messages to the signer
 using the [window.postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) API.
 
 ## Authentication
 
-* The relying party must authenticate the signer by the `origin` of the window that it opens.
-* The signer must authenticate the relying party by the `origin` property of the `message` event that it received.
+* The relying party must authenticate the signer by checking if both:
+  - The received message `source` property is equal to the `window` that was opened.
```

Comment:
Doesn't this defeat the purpose of redirecting? I.e. after a redirect, this will never match?

Reply:
As discussed in WG today, I wasn't aware that a message eventsource could be used instead of window.opener. I'll update this MR.